src/Makefile.m32

```diff
@@ -399,7 +399,7 @@ ifndef USE_LDAP_OPENLDAP
 curl_LDADD += -lwldap32
 endif
 endif
-curl_LDADD += -lws2_32
+curl_LDADD += -lws2_32 -lbcrypt
```

Comment:
So `bcrypt` would become an unconditional requirement to build curl? Wouldn't that mean dropping Windows XP support?